src/Concordium/Client/Output.hs

```diff
@@ -1088,7 +1090,58 @@ printChainParametersV1 ChainParameters {..} = tell [
+  [i|  + maximum factor a pool may stake relative to the baker's stake: #{_cpPoolParameters ^. ppLeverageBound}|],
+  [i|  + pool owner cooldown duration: #{durationToText (durationSeconds (_cpCooldownParameters ^. cpPoolOwnerCooldown) * 1000)}|],
+  [i|  + allowed range for finalization commission: #{showInclusiveRange show (_cpPoolParameters ^. (ppCommissionBounds . finalizationCommissionRange))}|],
+  [i|  + allowed range for baking commission: #{showInclusiveRange show (_cpPoolParameters ^. (ppCommissionBounds . bakingCommissionRange))}|],
+  [i|  + allowed range for transaction commission: #{showInclusiveRange show (_cpPoolParameters ^. (ppCommissionBounds . transactionCommissionRange))}|],
+  "",
+  [i|\# Passive delegation parameters:|],
+  [i|  + finalization commission: #{_cpPoolParameters ^. (ppPassiveCommissions . Types.finalizationCommission)}|],
+  [i|  + baking commission: #{_cpPoolParameters ^. (ppPassiveCommissions . Types.bakingCommission)}|],
+  [i|  + transaction commission: #{_cpPoolParameters ^. (ppPassiveCommissions . Types.transactionCommission)}|],
+  "",
+  [i|\# Parameters related to delegators: |],
+  [i|  + delegator cooldown duration: #{durationToText (durationSeconds (_cpCooldownParameters ^. cpDelegatorCooldown) * 1000)}|],
+  "",
+  [i|\# Exchange rate parameters: |],
+  [i|  - EUR per CCD rate (approx): #{printf "%.4f" (realToFrac (1000000 / _erMicroGTUPerEuro _cpExchangeRates) :: Double) :: String}|],
+  [i|  - EUR per Energy rate: #{showExchangeRate (_erEuroPerEnergy _cpExchangeRates)}|],
+  [i|  - microCCD per EUR rate: #{showExchangeRate (_erMicroGTUPerEuro _cpExchangeRates)}|],
+  "",
+  [i|\# Parameters that affect rewards distribution:|],
+  [i|  + mint amount per reward period: #{_cpTimeParameters ^. tpMintPerPayday}|],
+  [i|  + mint distribution:|],
+  [i|     * baking reward: #{_cpRewardParameters ^. mdBakingReward}|],
+  [i|     * finalization reward: #{_cpRewardParameters ^. mdFinalizationReward}|],
+  [i|  + transaction fee distribution:|],
+  [i|     * baker: #{_cpRewardParameters ^. tfdBaker}|],
+  [i|     * GAS account: #{_cpRewardParameters ^. tfdGASAccount}|],
+  [i|  + GAS rewards:|],
+  [i|     * baking a block: #{_cpRewardParameters ^. gasBaker}|],
+  [i|     * adding a finalization proof: #{_cpRewardParameters ^. gasFinalizationProof}|],
+  [i|     * adding a credential deployment: #{_cpRewardParameters ^. gasAccountCreation}|],
+  [i|     * adding a chain update: #{_cpRewardParameters ^. gasChainUpdate}|],
+  "",
+  [i|\# Time parameters:|],
+  [i|  + reward period length: #{_cpTimeParameters ^. tpRewardPeriodLength} epochs|],
+  "",
+  [i|\# Consensus parameters: |],
+  [i|  + timeout parameters: #{_cpConsensusParameters ^. cpTimeoutParameters}|],
+  [i|  + minimum block time: #{_cpConsensusParameters ^. cpMinBlockTime}|],
+  [i|  + block energy limit: #{_cpConsensusParameters ^. cpBlockEnergyLimit}|],
```

Comment:
Are you sure these display nicely? TimeoutParameter is a structure, no? So it won't exactly display nicely. Fix if correct.

Reply:
Thanks for pointing out. Fixed.